render-ws-spark-client/src/main/java/org/janelia/render/client/spark/pipeline/AlignmentPipelineParameters.java

```diff
@@ -50,6 +59,7 @@ public AlignmentPipelineParameters(final MultiProjectParameters multiProject,
                                        final MatchCopyParameters matchCopy,
                                        final AffineBlockSolverSetup affineBlockSolverSetup) {
```

Comment:
This hardcodes all existing steps, isn't it? Is there a way around that? I guess every approach that deals with those in a more generic (and dynamic) way probably loses type information, which would be a strong argument against doing that.

Reply:
Yeah - I was going for explicit declaration in a small number of places rather than a pure generic solution.